var/spack/repos/builtin/packages/mofem-users-modules/package.py

```diff
@@ -43,14 +43,28 @@ class MofemUsersModules(CMakePackage):
 
     homepage = "http://mofem.eng.gla.ac.uk"
     url = "https://bitbucket.org/likask/mofem-joseph/downloads/users_modules_dummy"
-    version('1.0', '5a8b22c9cdcad7bbad92b1590d55edb1', expand=False)
+
+    version('0.8.12', '5a8b22c9cdcad7bbad92b1590d55edb1', expand=False)
```

Comment:
Frankly, I am not a fan of these dummy versions that duplicate the main dependency. 

If users need to control which mofem-cephas to use, this shall be done on CLI or in packages.yaml

Reply:
Users modules is a git submodule of mofem core lib, it is not a dummy in the sense that which each version different code of user module is compiled. 

Currently is a bit of lack of consistency form my side, since the source is installed with git submodules with core lib, and package mofem-users-modules is used to compile this source code. 

I going to move correctly this with next pull request, that mofem-users-modules will work as a proper extension. However, I need to have this transition stage.

Reply:
Ok, sounds good to me